app/views/auth/CreateAccountView/CreateAccountForm.tsx

```diff
@@ -32,13 +34,16 @@ export const createAccountFormOnSubmit = async (
   helpers: FormikHelpers<CreateAccountFormValues>
 ): Promise<void> => {
   const { isMountedRef, createAccount } = pseudoProps;
-  const { accountName, password } = values;
+  const { accountName, password, checkedSavePassword } = values;
   const { setStatus, setErrors, setSubmitting } = helpers;
 
   try {
     await createAccount(accountName, password);
 
     if (isMountedRef.current) {
+      if (checkedSavePassword) {
+        ipcRenderer.sendSync('set-password', accountName, password);
+      }
```

Comment:
Ideally, instead of directly using ipcRenderer, it would call a callback provided by the parent component.